templates/activity.mustache

```diff
@@ -74,11 +74,26 @@
         {{> filter_siyavula/mastery}}
     {{/sectionid}}
 
-    <div id="monassis" class="monassis monassis--practice monassis--maths monassis--siyavula-api">
+    {{#assignmentid}}
+        <div id="monassis" class="monassis monassis--practice monassis--maths monassis--siyavula-api">
+            <div class="sv-session sv-session--assignment">
+                <div class="sv-session__title"> Assignment </div>
+                <div class="sv-session__actions">
+                    <div class="sv-session-badge--assignment"></div>
+                </div>
+            </div>
```

Comment:
I think you have some missing closing div tags here.

Reply:
HI @HAASLEWER I don't think I have, some of the div's are just closed on the same line, so it may look like closing tags are missing but they aren't